Source/Core/Core/ConfigManager.cpp

```diff
@@ -249,6 +249,8 @@ void SConfig::SaveCoreSettings(IniFile& ini)
   core->Set("OverrideGCLang", bOverrideGCLanguage);
   core->Set("DPL2Decoder", bDPL2Decoder);
   core->Set("Latency", iLatency);
+  core->Set("AudioStretch", m_audio_stretch);
```

Comment:
Reminder that maintaining compatibility with unsupported forks isn't something to strive for.